Provenance-Emu/Provenance
# Update Reachability.swift to 5.0.0

### What does this PR do
Update `Reachability`

### How should this be manually tested
Make sure a clean build works, no errors due to incompatible `Reachability`-callsites

### Any background context you want to provide
It seems like `Reachability` was updated in https://github.com/Provenance-Emu/Provenance/commit/f5c0fccbef806effd72977325e189967aae3f9d2, but Carthage wasn't updated, giving build errors on current master, when setting up from scratch
